pkgs/development/libraries/libieee1284/default.nix

```
@@ -14,10 +14,10 @@ stdenv.mkDerivation rec {
   };
 
   patches = [
-    # Fix build on Musl.
-    (fetchpatch {
-      url = "https://raw.githubusercontent.com/void-linux/void-packages/861ac185a6b60134292ff93d40e40b5391d0aa8e/srcpkgs/libieee1284/patches/musl.patch";
-      sha256 = "03xivd6z7m51i5brlmzs60pjrlqyr4561qlnh182wa7rrm01x5y6";
+    (fetchurl {
+      name = "musl.patch";
```

It's a bit nicer to see the name of the package that patches relate to at a glance.
```suggestion
      name = "libieee1284-musl.patch";
```

---

I've never seen any other package in Nixpkgs do this.

---

You're right, most packages don't. I'd like they did though, and this is the kind of change that can be done incrementally, step-by-step over time.